lib/crowbar/init/helpers.rb

```diff
@@ -254,6 +254,18 @@ def update_config_db
         )
       end
 
+      def ensure_admin_node_exist
+        # temporary workaround to enable gating for crowbar-core#920
+        return true unless File.exist?("/opt/dell/crowbar_framework/app/models/chef_node.rb")
```

Comment:
I want to merge this now anyway, I need to rebase 2-3 times a day and then I often forget to add this patch to the gating. Which results in extra time I need to spend to fix the gating and see the real test result. I think this small condition doesn't hurt for now and makes the other gating more reliable.